packages/shared/src/lib/constants.ts

```diff
@@ -64,3 +62,8 @@ export const authUrl =
 export const heimdallUrl = isDevelopment
   ? process.env.NEXT_PUBLIC_HEIMDALL_URL || 'http://127.0.0.1:3000'
   : authUrl;
+
+export const rebrandly = {
+  bookmarkLoops: 'https://r.daily.dev/bookmarkloops',
+  migrateUserToStreaks: 'https://r.daily.dev/streaks',
+};
```

Comment:
This was discussed before, and we should probably do it similarly to image links, just so we will have to import one variable from the file. Either way works for me though.

Reply:
Code splitting wise it would be worse since the whole object would then be pulled all over the place. Non blocking I just wanted to make it as it is currently.

Reply:
I see. In that case, it is maybe better to keep them plain as you mentioned.